Install poetry

freeseacher opened the pull request:
Poetry is supported by Pycharm by default but its installation is a bit pain. That merge request simplify it

johanfleury commented:
Hey, thanks for the PR.

How did you generate this list of packages?

I just tried with [`flatpak-pip-generator`](https://github.com/flatpak/flatpak-builder-tools/tree/master/pip) and it seems that you’ve included some extra packages and some packages with outdated versions. Also I feel like having an external file rather than putting everything in the main file would cleaner.

Could you please try with `flatpak-pip-generator` and use include according to the tool’s README.

freeseacher commented:
oh, i've done it bymyself one by one. and first part of them are from "the internets" (c)
i will regenerate package list with tool above and update pr

freeseacher commented:
Switching to that tool also switched from whl dists to tar.gz and that required even more dependencies. 
and even more updates in previous stages. so i've updated them all :roll_eyes:

freeseacher commented:
@johanfleury up ?

johanfleury commented:
Hey, sorry, didn’t have time to review, will do this weekend or next week.

johanfleury commented on com.jetbrains.PyCharm-Professional.yaml:
```diff
@@ -20,34 +20,205 @@ modules:
+        sha256: 536b67a0697f2e4af32ab176c00a50ac2899c5a05e0d8e2dadac8e58888283f9
+
+
+#  - name: miniconda
```
Can you remove these commented lines please?

freeseacher replied:
yep done

johanfleury reviewed (changes requested):
I don’t have time to review this in details but apart from the commented lines, it seems good. Just makes this quick change and I’ll merge it.
